CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 * Fixed NPE problem happened in SharedRealm.finalize() (#3730).
 * Fixed a build error when the project is using Kotlin (#4087).
+* Fixed a bug that the classes were replaced with a class in Gradle's classpath (#3568).
```

Comment:
Perhaps `Fixed a bug causing classes to be replaced by classes already in Gradle's classpath` ?